openassessment/templates/openassessmentblock/staff_area/oa_student_info_assessment_detail.html

```diff
@@ -0,0 +1,57 @@
+{% load i18n %}
+{% spaceless %}
+<div class="staff-info__status ui-staff__content__section wrapper--ui--collapse staff-info__{{ class_type }}__assessments">
+    <div class="ui-staff ui-toggle-visibility is--collapsed">
+        <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
+            <i class="icon fa fa-caret-right" aria-hidden="true"></i>
+            <span>{% trans translated_title %}</span>
+        </h2>
+        <div class="ui-toggle-visibility__content">
+            {% for assessment in assessments %}
+                {% with assessment_num=forloop.counter %}
+                    {% if assessments|length > 1 %}
+                        <h4 class="title--sub">
+                            {% blocktrans %}
+                                Assessment {{ assessment_num }}: #TODO: Does this really need to very between "Peer {}:" and "Assessment {}:"? The latter seems cleaner to me in all cases, especially given we already have a translated_title above
```

Comment:
Note to eventual reviewers:

Here's a screenshot showing the difference between the 2 potential values of this string. I think it looks good as `Assessment N:` in both cases, how does everyone else feel?

<img width="852" alt="screen shot 2016-01-15 at 3 43 54 pm" src="https://cloud.githubusercontent.com/assets/7373924/12364825/f4afe82c-bb9e-11e5-8f0a-346d56beeac5.png">